RELEASING.md

```diff
@@ -0,0 +1,244 @@
+
+4. Release the appropriate [Jira version](https://tickets.metabrainz.org/projects/MBVM?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page);
+   as its description, set the git release URL (https://github.com/metabrainz/musicbrainz-docker/releases/tag/$version_number).
+   Archive the previous non-schema change releases, if not yet archived. Create a new version
+   for the next expected release, with the description "next release".
+
+5. Edit the blog post to link to the new musicbrainz-docker release.
+
+## Release beta
+
+It has some differences with the production release process; follow these steps:
+
+1. On the translations update step, do not just update translated messages,
+   also update source messages for translation. This involves four steps:
+
+   1. Run `./po/update_translations.sh --commit` to download the latest .po files
+      from Transifex and commit them to the `master` branch
+
+   2. Run `./po/update_pot.sh` to generate new .pot files from the
+      database and templates. It's often a good idea to manually check
+      the changes to the .pot files: this is a good moment to find typos
+      that were missed during code review, or small changes to lines that
+      don't seem useful enough to justify breaking the existing translations.
+      If you find any of these, you might want to correct the issues and
+      generate the files again. Once you're done, commit the changes.
+
+   3. Push the `.pot` files generated in step 2 to Transifex. To push only the
+      ones that changed, give a comma-separated list:
+
+      ```sh
+      cd po/
+      tx push -s -r musicbrainz.server,musicbrainz.instruments
+      ```
+
+      If you want to push languages, keep in mind you need to push
+      `musicbrainz.languages-9` (not `musicbrainz.languages`).
+
+   4. Push the commits from steps 1 & 2 to `master`.
+
+2. On the git branches merge step, to update the `beta` branch with the changes from the `master` branch,
+   merge `master` into `beta` (with `git merge --log=876423 --no-ff master`) and push to `beta`.
+   (Skip this, of course, if you're just deploying changes pushed directly to the `bet`a branch.)
+
+3. On the [build Docker images step](#build-docker-images),
+   enter `beta` for _IMAGE_BRANCH_ when doing “Build with Parameters”.
+   Wait until the build has completed.
+
+4. On the [deployment’s announcement step](#announce-the-deployment),
+   [set the banner message on beta](https://beta.musicbrainz.org/admin/banner/edit) to
+
+   ```html
+   Beta website is being updated, slowdowns may occur for a few minutes, thanks for your patience.
+   ```
+
+5. On the [deployment step](#deploy-to-production) itself,
+   run `./script/update_containers.sh beta`. Wait until the deployment has completed.
+
+   Notes:
+   - Background task runners follow the `production` branch only,
+     which means that new reports are not available on the beta website for now.
```

Comment:
It seems good to have the report testable by running it once by hand, so maybe:

```suggestion
   - Background task runners follow the `production` branch only,
     which means that new reports are not available on the beta website for now.
     If you want users to be able to test the report, you can run it once by hand
     to create a static version unless you run the report by hand from a beta container
     (see `[HACKING-PROD.md](HACKING-PROD.md)` for instructions).
```

And adding a small section to HACKING-PROD.md with the command to run?

Reply:
Oh! I see you gave instructions for this on https://github.com/metabrainz/syswiki/pull/44/ anyway :) Commenting there.

Reply:
Answered on it.